.github/workflows/maven.yml

```diff
@@ -21,4 +21,4 @@ jobs:
       with:
         java-version: 1.8
     - name: Build with Maven
-      run: mvn -B package --file src/backend/jag-efiling-api/pom.xml
+      run: mvn -B test --file src/backend/jag-efiling-api/pom.xml -P all
```

Comment:
#### Question
why were we running `mvn package` before? doesn't that generate a war/jar file? ie; take the compiled code and package it in its distributable format

Reply:
it was the default git action that I didn't change, then added the CI one yesterday. Alex pointed out that test would be the better option to run.

Reply:
yup makes sense! okay sounds good